src/app/components/Map/Map.tsx

```diff
@@ -22,12 +22,12 @@ export default function MapBox(): JSX.Element {
   const mapContainer = useRef<HTMLDivElement | null>(null)
   const map = useRef<null | Map>(null)
   const [distance, setDistance] = useState<number>(0)
-  const [location, setLocation] = useState<number[]>([])
+  const [location, setLocation] = useState<GeoJSON.Position | null>(null)
```

Comment:
It would be nice to use an import and just have the type set like this `useState<Position | null>`